### PR TITLE
[REF] test_pylint: Show warning of empty paths but no error. Add colors to pylint global result.

### PR DESCRIPTION
**Pull request by moylop260**
Change from this
captura de pantalla 2015-08-31 a las 1 41 39 p m

to this
captura de pantalla 2015-08-31 a las 1 40 46 p m

And add red message in error case.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks :+1:

File: travis/test_pylint [hunk 1]
```diff
@@ -22,11 +23,13 @@ expected_errors = int(
     os.environ.get('PYLINT_EXPECTED_ERRORS', 0))
 
 exit_status = 0
-if count_errors != expected_errors:
-    print("pylint expected errors {expected_errors}, "
+if count_errors == -1:
```
Comment (dreispt) [inline travis/test_pylint]: Use <0 instead?

**Comment by dreispt** [comment]
:+1: Thanks!

File: travis/test_pylint (hunk 1 above)
Comment (moylop260) [inline travis/test_pylint]: We have:
`-1: python modules not found`
-2 should be a unknow error.

Comment (moylop260) [inline travis/test_pylint]: Then, I'm adding a validation to get the case of `empty-folder` (-1)
I can't to use a global negative value to `empty-folder`
Maybe in future you add a new negative error:
`-2: Other handled error that no is empty folder`

**Comment by dreispt** [comment]
Can we fast-track this one?

**Comment by pedrobaeza** [comment]
Yeah, it's really needed

**Comment by dreispt** [comment]
:green_heart:

**Pull request merged**